src/main/webapp/studentHelp.jsp

```diff
@@ -0,0 +1,60 @@
+<%@ taglib tagdir="/WEB-INF/tags" prefix="t" %>
+<t:helpPage>
+    <h1>Help for Students</h1><br>
+    <h4 class="text-color-primary">Getting Started</h4>
+    <br>
+    <ol type="1">
+        <li>
+            <span class="text-bold">Join the course</span> by clicking the link given in the email you received from TEAMMATES.<br>
+            Note that you need to join a course only once per course.<br>
+            Also note that joining a course is optional. If you do not join, you can still submit feedback and view feedback responses using links sent to you by TEAMMATES.
+            If you join, you get access to extra features such as the ability to see all your feedback sessions in a single home page.<br>
+        </li>
+        <br>
+        <li>
+            <span class="text-bold">Do the pending submissions</span> (if any).
+        </li>
+        <br>
+        <li>
+            You can <span class="text-bold">edit your submissions</span> up to the closing time of the respective session.
+        </li>
+        <br>
+        <li>
+            You can <span class="text-bold">view session results</span> after the instructor has published session results.
+            The system will notify you via email when results are available for viewing.
+        </li>
+    </ol>
+    <hr>
+    <h4 class="text-color-primary">Frequently Asked Questions</h4>
+    <br>
+    <ul>
+        <li>
+            <div>
+                <b>What should I do if I cannot see any/some of my courses I should be able to see in the system?</b>
+            </div>
+            <div>
+                These are the possible reasons:
+                <ol>
+                    <li>
+                        You used a different Google account to access TEAMMATES in the past.
+                        In that case, you need to  use the same Google account to access TEAMMATES again.
+                        Logout and re-login using the other Google account.</li> 
```

Comment:
Maybe shift the `</li>` to the next line to be consistent with the next list item